source/_components/sensor.syncthru.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "Samsung SyncThru Printer"
+description: "Instructions how to integrate a Samsung printer providing SyncThru within Home Assistant."
+date: 2018-02-19 23:33
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: # where do I place this
```

Comment:
`/source/images/supported_brands`

Reply:
How do you handle copyright issues? Where to store information that we are allowed to use brand logos?